ASCII_SUBSEQUENCES/ASCII_SUBSEQUENCES.cpp

```diff
@@ -0,0 +1,25 @@
+#include<iostream>
+#include<string>
+#include<cmath>
+using namespace std;
+string str;

```

Comment:
Also give a space before and after this statement
string str;

Reply:
It means one line space before and after this statement string str;

Reply:
correct